### ISSUE TITLE
Pages explorer "close" button does not display

### ISSUE THREAD

**Issue by thibaudcolas** [wagtail/wagtail#5396]
### Issue Summary

Spotted in #5394. On mobile viewports, the page explorer menu is meant to display a "Close explorer" button:

<img width="374" alt="close-explorer-button" src="https://user-images.githubusercontent.com/877585/59695025-74843000-91e1-11e9-924b-f9c57d352d73.png">

This button isn’t displayed anymore – it seems that this is a regression from #3740, which changed the definition of `u-hidden` to rely on `!important`:

```diff
.u-hidden {
-    display: none;
+    display: none !important;
}
```

thereby overriding the corresponding code to display this menu on mobile: https://github.com/wagtail/wagtail/blob/3eb71d702e9ba779a532f132d9fb59dc7b140faa/client/src/components/Explorer/Explorer.scss#L72-L76.

### Steps to Reproduce

1. Log into the CMS admin on a mobile device
2. Open the Pages explorer menu
3. There should be a "Close explorer" menu visible.

Any other relevant information. For example, why do you consider this a bug and what did you expect to happen instead?

* I have confirmed that this issue can be reproduced as described on a fresh Wagtail project: yes

### Technical details

* Wagtail version: 2.6a0

---

I think an easy fix for this would be to rewrite the CSS to stop relying on `u-hidden` - but we might also want to completely change this anyway so there is always a Close button available no matter the screen size.

**Comment by thibaudcolas** [comment]
Fixed in #5773 by @saevarom – this will be released in Wagtail 2.9.
